common/config.go

```diff
@@ -43,12 +49,22 @@ type PluginConfig struct {
 	IpamConfig `toml:"ipam-config"`
 }
 
+func (pc *PluginConfig) String() string {
+	return fmt.Sprintf("{ConfigFile: %v, Debug: %v, GridConfig: %v, IpamConfig: %v}",
+		pc.ConfigFile, pc.Debug, pc.GridConfig, pc.IpamConfig)
+}
+
```

Comment:
Any reason why the receiver is pointer to PluginConfig?

Reply:
no reason,  It will work without pointer also. to make it consistent i will remove pointer as reciever.